cmd/agent/app/agent.go

```diff
@@ -153,14 +153,19 @@ func setupControllers(mgr controllerruntime.Manager, opts *options.Options, stop
 }
 
 func registerWithControlPlaneAPIServer(controlPlaneRestConfig *restclient.Config, memberClusterName string) error {
+	kubeClient := kubeclientset.NewForConfigOrDie(controlPlaneRestConfig)
 	karmadaClient := karmadaclientset.NewForConfigOrDie(controlPlaneRestConfig)
```

Comment:
Can we use `gclient` here?
`gclient.NewForConfigOrDie(controlPlaneRestConfig)`

Two clients seems a little bit confused.